python/pyspark/sql/tests/test_datasources.py

```diff
@@ -192,6 +193,23 @@ def test_ignore_column_of_all_nulls(self):
         finally:
             shutil.rmtree(path)
 
+    def test_jdbc(self):
+        db = f"memory:{uuid.uuid4()}"
```

Comment:
@ueshin What kind of config do I need to make this work? I got error: Database xxx not found.

Reply:
I think it should work without any configs.
Could you try to rebuild?

Reply:
@ueshin Yeah, the python test works. I want to do the same for scala client, but I am missing some spark config?

Reply:
Does #40358 include the test? I can try it and see.

Reply:
Oh I missed "create", thanks. Works for scala client too!